library/L1_Peripheral/lpc40xx/adc.hpp

```diff
@@ -190,6 +190,9 @@ class Adc final : public sjsu::Adc
   /// largest value it can have is 2^12 = 4096
   static constexpr uint8_t kActiveBits = 12;
 
+  /// The LPC40xx ADC has a reference voltage of 3.3V.
+  static constexpr units::voltage::microvolt_t kReferenceVoltage = 3.3_V;
```

Comment:
It would make sense for the reference voltage for the ADC to come from a constructor. Having the parameter have a default value of 3v3 is good, but users of the lpc40xx ADC may choose a different reference voltage than 3v3 depending on their application.